src/fastfetch.c

```diff
@@ -840,25 +841,21 @@ static void writeConfigFile(FFdata* data, const FFstrbuf* filename)
     ffOptionsGenerateLibraryJsonConfig(&instance.config.library, doc);
     ffMigrateCommandOptionToJsonc(data, doc);
 
-    if (ffStrbufEqualS(filename, "-"))
-        yyjson_mut_write_fp(stdout, doc, YYJSON_WRITE_INF_AND_NAN_AS_NULL | YYJSON_WRITE_PRETTY_TWO_SPACES, NULL, NULL);
-    else
+    FF_AUTO_CLOSE_FILE FILE *fp = stdout;
```

Comment:
Seems you are attempting to close stdout. Don't do that.

Reply:
Done.